File: demo-android/app/src/test/java/com/example/demo/testing/TaskFixtures.kt
package com.example.demo.testing

import com.example.demo.data.model.Task
import com.example.demo.data.model.TaskPriority
import com.example.demo.data.model.TaskRequest
import com.example.demo.data.model.TaskStatus

object TaskFixtures {
    val sampleTask = Task(
        id = "task-1",
        title = "Buy milk",
        description = "2%",
        status = TaskStatus.TODO,
        priority = TaskPriority.MEDIUM,
        createdDate = "2024-01-01T10:00:00Z",
        updatedDate = "2024-01-02T10:00:00Z"
    )

    val sampleRequest = TaskRequest(
        title = "Buy milk",
        description = "2%",
        status = TaskStatus.TODO,
        priority = TaskPriority.MEDIUM
    )
}

File: demo-react-native/src/test-utils/taskFixtures.ts
import {Task, TaskPriority, TaskStatus} from '@/data/models/task';

export const mockTask: Task = {
  id: 'task-1',
  title: 'Sample task',
  description: 'Task description',
  status: TaskStatus.TODO,
  priority: TaskPriority.MEDIUM,
  createdDate: '2026-06-17T10:00:00.000Z',
  updatedDate: '2026-06-17T12:00:00.000Z',
};
